doc/general_documentation/INSTALL_kinetic.md

```diff
@@ -0,0 +1,134 @@
+\page install-kinetic Install in Ubuntu 16.04 / ROS kinetic
+
+# Usage instructions for non-NASA users
+
+Install the 64-bit version of [Ubuntu 16.04](http://releases.ubuntu.com/16.04)
+on a host machine, and make sure that you can checkout and build code.
+If you are using a virtual machine, please use VMware. Virtualbox doesn't
+support some of our Gazebo plugins.
+
+    sudo apt-get install build-essential git
+
+*Note: You will need 4 GBs of RAM to compile the software. If you don't have
+that much RAM available, please use swap space.*
+
+*Note: Please ensure you install Ubuntu 16.04. Running the simulation in Ubuntu 18.04 is possible, but not officially supported, instructions in README_Ubuntu18.md. At this time we do not support
```

Comment:
add link to 18 instructions?

Reply:
This has been changed, I merged it with the melodic install instructions as suggested below